Patch by Robert Scheck <robert@fedoraproject.org> for libnet >= 1.0.2a, which adds
the missing quotations needed by GCC 3.3 and newer.

--- Libnet-1.0.2a/include/libnet.h		2001-02-07 00:19:25.000000000 +0100
+++ Libnet-1.0.2a/include/libnet.h.gcc33	2009-04-18 19:37:20.000000000 +0200
@@ -84,9 +84,9 @@
 #define LIBNET_VERSION  "1.0.2a"
 
 #if (!LIBNET_LIL_ENDIAN && !LIBNET_BIG_ENDIAN)
-#error "byte order has not been specified, you'll
-need to #define either LIBNET_LIL_ENDIAN or LIBNET_BIG_ENDIAN.  See the
-documentation regarding the libnet-config script."
+#error "byte order has not been specified, you'll"
+"need to #define either LIBNET_LIL_ENDIAN or LIBNET_BIG_ENDIAN.  See the"
+"documentation regarding the libnet-config script."
 #endif
 
 #endif  /* __LIBNET_H */
